Mobile Buy SDK/Mobile Buy SDK/Data/BUYClient+Routing.h

```diff
@@ -36,13 +36,13 @@
 - (NSURL *)urlForCollectionListingsWithParameters:(NSDictionary *)parameters;
 
 - (NSURL *)urlForCheckouts;
-- (NSURL *)urlForCheckoutsWithToken:(NSString *)token;
-- (NSURL *)urlForCheckoutsProcessingWithToken:(NSString *)token;
-- (NSURL *)urlForCheckoutsCompletionWithToken:(NSString *)token;
-- (NSURL *)urlForCheckoutsShippingRatesWithToken:(NSString *)token parameters:(NSDictionary *)parameters;
+- (NSURL *)urlForCheckoutsForToken:(NSString *)token;
```

Comment:
Agreed. Would be nice to make this more natural sounding, but this is private and we can do so at any time in the future.